src/applications/disability-benefits/686c-674/config/chapters/674/student-name-ssn/studentNameSSN.js

```diff
@@ -55,6 +55,10 @@ export const uiSchema = {
           isChapterFieldRequired(formData, 'report674'),
       },
     },
+    isParentOrGuardian: {
+      'ui:title': 'Is this your child, or are you their guardian?',
```

Comment:
The wording of this question feels sort of awkward. Feels like it'd be better to ask something like `Are you either a parent or guardian to this child?`

Reply:
![](https://thumbs.gfycat.com/EminentTinyAlbino.webp)

Reply:
@micahchiang or just `are you a parent or guardian of this child?` @jason-gcio what do you think?

Reply:
Nice catch